azure-pipelines.yaml

```diff
@@ -0,0 +1,121 @@
+trigger:
+  tags:
+    include:
+      - v*
+  branches:
+    include:
+    - master
+  # test and upload my code as an artifact named WebSite
+jobs:
+- job: Build
+  strategy:
+    matrix:
+      mac_36:
+        imageName: 'macOS-10.14'
+        python.version: '3.6'
+      mac_37:
+        imageName: 'macOS-10.14'
+        python.version: '3.7'
+      linux_py36:
+        imageName: 'ubuntu-16.04'
+        python.version: '3.6'
+      linux_py37:
+        imageName: 'ubuntu-16.04'
+        python.version: '3.7'
+  pool:
+    vmImage: $(imageName)
+
+  steps:
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '$(python.version)'
+      architecture: 'x64'
+
+  - script: brew install node@12
+    displayName: 'Install NodeJS'
```

Comment:
brew commands should be limited only to osx

Reply:
that's true i've added a condition `condition: eq(variables['imageName'], 'macOS-10.14')` to install with brew and for ubuntu i added another script

Reply:
Use `Agent.OS` to prevent this from breaking when we one day upgrade to `10.15`